src/main/java/io/antmedia/datastore/db/types/Broadcast.java

```diff
@@ -207,6 +207,18 @@ public class Broadcast {
 	@ApiModelProperty(value = "Initial time to start playing. It can be used in VoD file or stream sources that has seek support")
 	private long seekTimeInMs = 0;
 
+	@ApiModelProperty(value = "The list of the play only users who requested to publish the stream. It is used in conference scenario.")
+	List<String> publisherRequestList = new ArrayList<String>();
+
+	@ApiModelProperty(value = "The list of the users who are approved to publish the stream. It is used in conference scenario.")
+	List<String> publisherFromListenerList = new ArrayList<String>();
```

Comment:
Do we need to have different fields for user scenarios? I mean publisherFromListenerList seems like scenario specific to me.